frappe/public/build.json

```diff
@@ -153,7 +153,8 @@
 		"public/js/lib/leaflet/leaflet.js",
 		"public/js/lib/leaflet/leaflet.draw.js",
 		"public/js/lib/leaflet/L.Control.Locate.js",
-		"public/js/lib/leaflet/easy-button.js"
+		"public/js/lib/leaflet/easy-button.js",
+		"node_modules/qz-tray/qz-tray.js"
```

Comment:
can we load this on demand, with `frappe.require` ?

Reply:
yes, that makes sense! didn't know `frappe.require` existed...

`qz-tray.js` and `sha256.min.js` are now loaded only when calling the qz.connect function.